theano/sandbox/solve.py

```diff
@@ -1,75 +1,7 @@
-from __future__ import absolute_import, print_function, division
+import warnings
+from theano.tensor.slinalg import solve
```

Comment:
You need to add this file to the whitelist in theano/tests/test_flake8, otherwise it generates an unused import error.